docs/api.rst

```diff
@@ -61,6 +61,7 @@ The computational backends that :code:`pyhf` provides interfacing for the vector
    numpy_backend.numpy_backend
    pytorch_backend.pytorch_backend
    tensorflow_backend.tensorflow_backend
+   common.TensorViewer
```

Comment:
`TensorViewer` is in the public API and so should be documented as such. I actually think that it would be better to have this documentation be its own PR, but I thought I'd raise it here and if you agree then this can get dropped and it can get spun off into its own thing.

Alternatively, does `TensorViewer` need to be in the public API? If not, then can we just rename it to `_TensorViewer` and document it in the developer API later?

Reply:
sounds good

Reply:
Yeah, I renamed it to `_TensorViewer` in https://github.com/diana-hep/pyhf/pull/556/commits/77be0d307a6b1706009a1100efe4bcdc606381cb (as you've already seen).